docs/src/technik/coding_conventions/tests_naming.md

```diff
@@ -0,0 +1,62 @@
+# Naming Convention für Tests
+
+## Kontext
+
+Damit der Code übersichtlicher und einheitlicher ist, sollen Naming Conventions eingesetzt werden. Damit wird für Tests gewährleistet, dass deren Inhalt klar ist, ohne den Code lesen zu müssen.
+
+## Entscheidung
+
+Die Bezeichnungen sollen dem Schema `should_<result>_when_<input>` folgen, wobei Result (= ExpectedBehavior, bzw. erwartetes Ergebnis) und Input (= StateUnderTest, bzw. zu testender Zustand) in CamelCase gehalten werden. Dem Schema entsprechend sind die Testnamen auch auf englisch zu formulieren.
+
+Wir haben uns darauf geeinigt, die zu testenden Methoden mit `@Nested` zu gruppieren. Im Fall von überladenen Methoden werden diese innerhalb der Methodenklasse zusätzlich verschachtelt und ebenfalls mit `@Nested` annotiert.
+
+### Beispiele
+
+```java
+//   should_expectedResultInCamelCase_when_inputInCamelCase() {}
+void should_returnDTO_when_givenValidId() {}
+void should_notThrowException_when_newDataSaved() {}
+void should_throwAccessDeniedException_whenAuthoritiesMissing() {}
+```
+
+##### Gruppierung überladener Methoden:
+Vereinfachter Pseudocode! Beispiel aus Vorfälle und Vorkommnisse Service
+```java
+@Mapper
+public interface EreignisModelMapper {
+
+  Ereignis toEntity(EreignisModel model);    
+    
+  List<Ereignis> toEntity(EreignisseWriteModel model);  
+}
+```
+
+```java{2-3,5-6,11-12} 
+class EreignisModelMapperTest {
+  @Nested
+  class ToEntity {                      // Name der zu testenden Methode
+    
+     @Nested
+     class ToEreignisEntity {           // nested overload 1
+         @Test
+         void should_returnEreignis_when_givenEreignisModel() {}
+     }
+  
+     @Nested
+     class ToListOfEreignisEntity {     // nested overload 2
+         @Test
+         void should_returnListOfEreignis_when_givenEreignisseWriteModel() {}
+     }
+  }
+}
+```
```

Comment:
Was hälst du davon das als [Code-Group](https://vitepress.dev/guide/markdown#code-groups).

Vorteil ist man sieht die beiden Dateinamen, Nachteil ist man kann nie beides zur selben Zeit sehen.

Reply:
siehe commit [b965b0a](https://github.com/it-at-m/Wahllokalsystem/pull/427/commits/b965b0a465a5c8c24d1a611666ee96bc340a6167)

Die Gruppierung finde ich gut. Damit der relevante Code im Fokus ist habe ich aber die Reihenfolge getauscht und der Test wird zuerst angezeigt.